<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseModulePrototypeAutoTile</class>
 <widget class="QWidget" name="DatabaseModulePrototypeAutoTile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>196</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>auto-tile-set</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QGroupBox" name="m_pBasicSet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>171</height>
         </size>
        </property>
        <property name="title">
         <string>basic set:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="m_pInnerCenter" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>51</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="database::ui::module::PrototypeBase" name="m_pModuleBase" native="true">
     <zorder>groupBox</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>database::ui::module::PrototypeBase</class>
   <extends>QWidget</extends>
   <header>databasemoduleprototypebase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
